elastalert/elastalert.py

```diff
@@ -628,7 +628,7 @@ def run_query(self, rule, start=None, end=None, scroll=False):
         if end is None:
             end = ts_now()
 
-        if rule.get('query_timezone') != "":
+        if rule.get('query_timezone') is not None:
```

Comment:
Would it be better to check for both None OR ""?

Reply:
I suspect that for most people the field will be undefined in the YAML, so would evaluate to None.

However it's a simple change to check for both, and it might catch some edge cases, so I've pushed the change. 👍